purge.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+""" purge.py
+    MediaWiki Action API Code Samples
+    Demo of `purge` module: Sending post request to purge cache of one page
```

Comment:
This demo is for post request to purge first 10 pages in the main namespace and not one page. On other note, you could consider adding another file to show how to purge one or two pages by specifying titles.

Reply:
Okay. Working on it